_includes/experience.html

```diff
@@ -19,6 +19,11 @@ <h3 class="job-title">{{ experience.role }}</h3>
     {% if experience.details %}
     <div class="details">
       <p>{{ experience.details }}</p>
+      <ul>
```

Comment:
@daniel-lanciana  I like the idea of bullet points but please keep it inside an if statement. That way it will be optional.

Reply:
Done.